tests/catch/unit/memory/hipArrayCommon.hh

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Wouldn't this make more sense to live in ```/tests/catch/include``` ?

Reply:
I would prefer to keep this header local to the array tests, like streamCommon here https://github.com/ROCm-Developer-Tools/HIP/tree/develop/tests/catch/unit/stream